### PR TITLE
refactor: clone: remove unnecessary lex complexity

### PR DESCRIPTION
**Pull request by katcipis**
# Reason for This Change

Simplify the stack clone code. No need to use lex/tokens for simple strings.

## Description of Changes

Just use cty.Value instead of lex when setting the new stack ID.

### CODE REVIEW

File: stack/clone.go [hunk 1]
```diff
@@ -153,16 +152,14 @@ func updateStackID(stackdir string) error {
 
 	logger.Trace().Msg("finding file containing stack definition")
 
-	stackFilePath, body := getStackBody(parser)
-	if body == nil {
-		return errors.E("updating stack ID: stack block not found")
+	stackFilePath := getStackFilepath(parser)
```
Comment (i4ki) [inline stack/clone.go:155]: maybe call this `stackParsedFile`?

**Review by i4ki — APPROVED** [review]
lgtm

File: stack/clone.go (hunk 1 above)
Comment (katcipis) [reply stack/clone.go:155]: hmm but it is not returning a parsed file :thinking: . Maybe stackBlockFilepath ? wanted to make it clear that it returns the filepath of where the stack block is defined. What is being "get" is kinda hard to explain with a small name :laughing:

Comment (i4ki) [reply stack/clone.go:155]: gotcha! I thought it was returning the hcl.File object.

**Pull request merged**